### PR TITLE
Update artifact-ignores.properties to deprecate plugin

### PR DESCRIPTION
**Pull request by sn-priya-jagyasi**
Update artifact-ignores.properties to deprecate the plugin "lightstep-incident-response-plugin"

### CODE REVIEW

File: resources/artifact-ignores.properties [hunk 1]
```diff
@@ -809,3 +809,4 @@ xframium = https://www.jenkins.io/security/plugins/#suspensions
 
 # https://jenkins.io/security/advisory/2022-11-15/
 config-rotator = https://www.jenkins.io/security/plugins/#suspensions
+lightstep-incident-response-plugin = https://github.com/jenkinsci/lightstep-incident-response-plugin/blob/main/README.md
```
Comment (daniel-beck) [inline resources/artifact-ignores.properties:812]:
```suggestion
lightstep-incident-response = https://github.com/jenkinsci/lightstep-incident-response-plugin/blob/main/README.md
```

Please do not ignore review feedback when opening a successor PR: https://github.com/jenkins-infra/update-center2/pull/665#discussion_r1047855084

**Review by daniel-beck — CHANGES_REQUESTED** [review]

**Comment by daniel-beck** [comment]
Please also choose a more appropriate location for the ignore list entry than in a section related to a past security advisory.

File: resources/artifact-ignores.properties (hunk 1 above)
Comment (sn-priya-jagyasi) [reply resources/artifact-ignores.properties]: cam someone review this please?

Comment (sn-priya-jagyasi) [reply resources/artifact-ignores.properties]: @daniel-beck Gentle Reminder.

Comment (zbynek) [reply resources/artifact-ignores.properties]: @sn-priya-jagyasi you still need to replace `lightstep-incident-response-plugin` with `lightstep-incident-response` as suggested both here and in the original PR.

**Pull request closed without merging**